cloudinit/net/openbsd.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit import log as logging
+from cloudinit import util
+import cloudinit.net.bsd
+
+LOG = logging.getLogger(__name__)
+
+
+class Renderer(cloudinit.net.bsd.BSDRenderer):
+
+    def __init__(self, config=None):
+        super(Renderer, self).__init__()
+
+    def write_config(self):
+        for device_name, v in self.interface_configurations.items():
+            if_file = 'etc/hostname.%s' % device_name
+            fn = util.target_path(self.target, if_file)
+            with open(fn, 'w') as fd:
+                if isinstance(v, dict):
+                    fd.write(v.get('address') + ' netmask '
+                             + v.get('netmask') + '\n')
+                else:
+                    fd.write('dhcp')
+
+    def start_services(self, run=False):
+        if not self._postcmds:
+            LOG.debug("netbsd generate postcmd disabled")
+            return
+        util.subp(['sh', '/etc/netstart'], capture=True)
```

Comment:
is /etc/netstart not already executable?

Reply:
This still needs a response.

Reply:
Nope, it's not:

```
openbsd-66$ ls -l /etc/netstart                                                                                                                                                              
-rw-r--r--  1 root  wheel  9580 Oct 12 10:34 /etc/netstart
```